recipes/recompile.rb

```diff
@@ -23,9 +23,7 @@
 ext_dir_prefix = node['php']['ext_dir'] ? "EXTENSION_DIR=#{node['php']['ext_dir']}" : ''
 
 node['php']['src_deps'].each do |pkg|
```

Comment:
Sam here we should be able to just pass in the array of packages. It's way faster